app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/further_return_check_tax_due.scala.html

```diff
@@ -40,6 +40,7 @@
     backLink: Call,
     yearToDateLiability: AmountInPence,
     previousYearToDateLiability: AmountInPence,
+    taxOwnedOnOriginalReturn: AmountInPence,
```

Comment:
the content should really only change for an amend return journey so it would make more sense if this was optional - if it's a new further return, then it would be `None`. For a further return, the original content should be there - otherwise if it's an amend return we want the new calculation content